pkg/resources/marketplace/manifestHelper_test.go

```diff
@@ -70,17 +70,17 @@ func TestGetCurrentCSVFromManifest(t *testing.T) {
                                 channels:
                                 - name: integreatly
                                 currentCSV: 3scale-operator.v1.9-7`,
-			ExpectedVer: "1.9.7",
+			ExpectedVer: "1.9-7",
```

Comment:
What was the reason for this change? 🤔

Reply:
The reason for this change was that originally the `-` sign was always being replaced by a `.` sign which was not what we expect with formats like `3scale-operator.v0.8.3-0.1649688682.p`. There was never a version like `1.9-7` in the history, but this function can be updated in the future to account for such cases.